src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx

```diff
@@ -187,7 +191,10 @@ class EstimateYourBenefitsForm extends React.Component {
 
   handleInternationalCheckboxFocus = e => {
     if (window.innerWidth <= SMALL_SCREEN_WIDTH) {
-      e.target.scrollIntoView();
+      const field = document.getElementById('classesOutsideUS');
+      if (field) {
+        scrollAndFocus(field);
```

Comment:
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/utilities/ui/index.js#L60